EntAppProject/src/main/resources/static/css/JobBookLogin.css

```diff
@@ -0,0 +1,420 @@
+
```

Comment:
This is a new css page that I created to server styles for the login page. This is part two of my css restructuring recommendation. In addition to making styles more specific to the purpose that they fulfill. I believe in fragmenting CSS styles in a similar way to how HTML files are often fragmented into an `index.html` and then page specific fragments. I suggest that the team keeps `JobBook.css` as the main CSS stylesheet that applies to all pages of the application. Then, for each individual page, the team should create a Stylesheet specific for that HTML page with new styles that either add styling or overwrite the existing styling to make it page appropriate. For example, `JobBook.css` may have a style that says
```css
h1 { color: orange; }
```
This will make all h1 tags orange by default. Then in the `LoginPage.css` file, if the team wants the style to be different, they can overwrite it to be something else. While this same outcome could be accomplished using classes or ids on the markups and on Stylesheet as I mentioned in the previous comment, multiple files for each page makes the use and purpose of each style very clear, and it makes reading the code far easier. Any style that is related to login will be in the login css file, which means if I want to change the login styles, I just look in the login stylesheet, which is far easier than digging through single stylesheet with hundreds of styles. Of course, this all becomes irrelevant if you choose to buy a UI template and just use that.